### DISCUSSION TITLE
Spectator challenges

### DISCUSSION THREAD

**Discussion by smoogipoo** [ppy/osu#15170] [Ideas] [answered]
A similar vibe to https://github.com/ppy/osu/discussions/15158 and https://github.com/ppy/osu/discussions/15061, but both of those have been answered and/or are going off in a different direction.

One idea brought up in the latest community meeting is spectator challenges to allow things like turning on/off mods and changing beatmap settings in real time. I think this would be quite fun, and could also potentially integrate (along with https://github.com/ppy/osu/pull/12274) with the Twitch Channel Points API.

**Comment by integerrr** [comment, marked as answer]
also #15151

**Reply by smoogipoo** [reply]
Ah, I saw that one too and forgot about it. Thanks!
